### PR TITLE
fix(curriculum): remove mention of null from challenge

### LINKED ISSUE #40738: Concept mentionned when its not been covered in previous challenges
**Describe your problem and how to reproduce it:**
Hi everyone,
In the challenge linked below, it is mentionned : 

> So far, you've seen how to use `if/else`, `&&,` `null` and the ternary operator (`condition ? expressionIfTrue : expressionIfFalse`) to make conditional decisions about what to render and when.

I have seen challenges about if/else, && and the ternary operator but not about null.

After looking for some documentation, it appears that it is still a valid method for hiding a component.

**Potential solutions :**

- I think a challenge could be added to introduce this concept.
- An other option would be to modify the [if/else challenge](https://www.freecodecamp.org/learn/front-end-libraries/react/render-with-an-if-else-condition) to cover this concept.
- Finally, a last option would be to delete the mention of this concept in the challenge linked below.

Any opinions ?

**Add a Link to the page with the problem:**
[React: Render Conditionally from Props](https://www.freecodecamp.org/learn/front-end-libraries/react/render-conditionally-from-props)

### PR DESCRIPTION
**Pull request by jblarriviere**
Delete the mention of campers having learnt about `null` for conditional rendering in the React curriculum as discussed in issue #40738 

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40738

### CODE REVIEW

File: curriculum/challenges/english/03-front-end-libraries/react/render-conditionally-from-props.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ dashedName: render-conditionally-from-props
 
 # --description--
 
-So far, you've seen how to use `if/else`, `&&,` `null` and the ternary operator (`condition ? expressionIfTrue : expressionIfFalse`) to make conditional decisions about what to render and when. However, there's one important topic left to discuss that lets you combine any or all of these concepts with another powerful React feature: props. Using props to conditionally render code is very common with React developers — that is, they use the value of a given prop to automatically make decisions about what to render.
+So far, you've seen how to use `if/else`, `&& and the ternary operator (`condition ? expressionIfTrue : expressionIfFalse`) to make conditional decisions about what to render and when. However, there's one important topic left to discuss that lets you combine any or all of these concepts with another powerful React feature: props. Using props to conditionally render code is very common with React developers — that is, they use the value of a given prop to automatically make decisions about what to render.
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/03-front-end-libraries/react/render-conditionally-from-props.md:11]:
```suggestion
So far, you've seen how to use `if/else`, `&&`, and the ternary operator (`condition ? expressionIfTrue : expressionIfFalse`) to make conditional decisions about what to render and when. However, there's one important topic left to discuss that lets you combine any or all of these concepts with another powerful React feature: props. Using props to conditionally render code is very common with React developers — that is, they use the value of a given prop to automatically make decisions about what to render.
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Thank you, for this PR. Be careful with the syntax used ⬇️

**Comment by jb-larriviere** [comment]
@ShaunSHamilton thanks for the review, i'm in the process of setting up FCC localy, so i should be able to spot that kind of mistakes later on
I've committed your suggestion

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍

**Pull request merged**